test/www/jxcore/bv_tests/testNativeMethod.js

```diff
@@ -2,83 +2,73 @@
 
 var platform = require('thali/NextGeneration/utils/platform');
 
-if (platform._isRealMobile) {
+var tape = require('../lib/thaliTape');
+var thaliMobileNativeWrapper = require('../node_modules/thali/NextGeneration/thaliMobileNativeWrapper');
 
-  var tape = require('../lib/thaliTape');
-  var thaliMobileNativeWrapper = require('../node_modules/thali/NextGeneration/thaliMobileNativeWrapper');
+var callbackPeer;
 
-  var callbackPeer;
-  var timeout;
+var test = tape({
+  setup: function (t) {
+    t.end();
+  },
+  teardown: function (t) {
+    thaliMobileNativeWrapper._registerToNative();
+    t.end();
+  }
+});
 
-  var test = tape({
-    setup: function (t) {
-      timeout = setTimeout(function () {
-        t.fail('No callback after calling registerToNative');
-      }, 5000);
-      t.end();
-    },
-    teardown: function (t) {
-      thaliMobileNativeWrapper._registerToNative();
-      t.end();
-    }
-  });
+test('onPeerLost calls jxcore',
+  function () {
+    return !platform._isRealMobile;
+  },
+  function (t) {
+    Mobile('peerAvailabilityChanged').registerToNative(function (peers) {
+      if (typeof peers.forEach !== 'function') {
```

Comment:
It would be more natural to use `Array.isArray` to check if peers is an array. The same for the second test below